### PR TITLE
Make onboarding + fix duplicate errors

### PR DESCRIPTION
**Pull request by patrickfeltes**
- Fix onboarding activity (now swipeable with tab indicator)
- Also fixed issues with duplicate resource errors (changed how google maps API key is imported)

To update your local version, put a value of MAPS_API_KEY=....

in local.properties file of the project. This will not go to source control since it is .gitignored

### CODE REVIEW

**Comment by patrickfeltes** [comment]
don't merge this yet, need to update actions to have local.properties

**Comment by patrickfeltes** [comment]
will rebase once https://github.com/HackIllinois/android/pull/366 is merged

**Comment by patrickfeltes** [comment]
nvm just doing it on this pr

File: app/src/main/java/org/hackillinois/android/view/onboarding/OnboardingActivity.kt [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        R.string.onboarding_scan_description,
+        R.string.onboarding_profile_description,
+        R.string.onboarding_leaderboard_description,
+    )
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_onboarding)
+
+        view_pager.adapter = ScreenSlidePagerAdapter(this)
```
Comment (timot3) [inline app/src/main/java/org/hackillinois/android/view/onboarding/OnboardingActivity.kt:46]: wow, didnt know of this

**Review by timot3 — APPROVED** [review]

**Pull request merged**